signer/src/transaction_signer.rs

```diff
@@ -1047,14 +1097,22 @@ where
             "🔐 creating now FROST coordinator to track DKG verification signing round"
         );
 
-        FrostCoordinator::load(
+        let coordinator = FrostCoordinator::load(
             storage,
             aggregate_key,
             signing_set,
             dkg_shares.signature_share_threshold,
             signer_private_key,
         )
-        .await
+        .await?;
+
+        let state_machine = dkg::verification::StateMachine::new(
+            coordinator,
+            aggregate_key,
+            Duration::from_secs(30), // TODO: Config?
```

Comment:
Can we set this to unlimited or none? Currently we rely on bitcoin blocks and/or other timeouts, having this may conflict with something else

Reply:
See below comment, they kinda hang together

Reply:
Added support for `None` = `unlimited` and use that in the creation in tx-signer. [85aae81](https://github.com/stacks-network/sbtc/pull/1324/commits/85aae816747a687f1c09e9819204f4c382363152)